api/app/services/relatedHelper.py

```diff
@@ -0,0 +1,89 @@
+from app.settings import settings
+import hnswlib
+import csv
+
+
+class RelatedHelper:
```

Comment:
maybe can should name this `NearestNeighbourSearcher` or `RelatedSearcher`?

Reply:
`RelatedSearcher` for now.